content/04.results.md

```diff
@@ -1,38 +1,38 @@
-Our analysis of the full text release of bioRxiv confirms this previous finding (Figure {@fig:biorxiv_categories}).
-The author-selected topic area abundances that we found in the full text largely matched previous studies [@doi:10.7554/eLife.45133; @doi:10.1001/jama.2017.21168].
-One exception was microbiology, which has a larger share of preprints than in a previous report from 2018 [@doi:10.7554/eLife.45133] (Figure {@fig:biorxiv_categories}).
+The preprint landscape is rapidly changing, and the number of bioRxiv preprints in our data download (71,118) was nearly double that of a recent study that reported on a snapshot with 37,648 preprints [@doi:10.7554/eLife.45133].
+Because the rate of change is rapid, we first analyzed category data and compared our results with previous findings.
+As in previous reports [@doi:10.7554/eLife.45133], neuroscience remains the most common category of preprint followed by bioinformatics (Figure {@fig:biorxiv_categories}).
+Microbiology, which was fifth in the most recent report [@doi:10.7554/eLife.45133], has now surpassed evolutionary biology and genomics to move into third.
 When authors upload their preprints, they select from three result category types: new results, confirmatory results or contradictory results.
-We found that nearly all preprints were categorized as new results, which is consistent with previous findings [@doi:10.1001/jama.2017.21168].
-
-#### Global Comparison of bioRxiv and PubMed Central 
-
-| Metric                | bioRxiv     | PMC           | NYTAC         | 
-|-----------------------|-------------|---------------|---------------| 
-| document count             | 71,118      | 1,977,647     | 1,855,658     | 
-| sentence count             | 22,195,739  | 480,489,811   | 72,171,037    | 
-| token count                | 420,969,930 | 8,597,101,167 | 1,218,673,384 | 
-| stopword count            | 158,429,441 | 3,153,077,263 | 559,391,073   | 
-| avg. document length        | 312.10      | 242.96        | 38.89         | 
-| avg. sentence length        | 22.71       | 21.46         | 19.89         | 
-| negatives                  | 1,148,382   | 24,928,801    | 7,272,401     | 
-| coordinating conjunctions  | 14,295,736  | 307,082,313   | 38,730,053    | 
-| coordinating conjunctions% | 3.40%       | 3.57%         | 3.18%         | 
-| pronouns                   | 4,604,432   | 74,994,125    | 46,712,553    | 
-| pronouns%                  | 1.09%        | 0.87%         | 3.83%         | 
-| passives                   | 15,012,441  | 342,407,363   | 19,472,053    | 
-| passive%                   | 3.57%       | 3.98%         | 1.60%         | 
+We found that nearly all preprints were categorized as new results, which is consistent with previous findings [@doi:10.1001/jama.2017.21168]. <!-- I cannot find this assertion or these numbers in this paper - where did you see it? -->

```

Comment:
I originally had a figure that showed majority of new results were in the bioRxiv repository, but Larry told me to remove the figure (as it was a waste of space) and just state majority of preprints were new results.

Reply:
Oh - I meant in the paper that the reference goes to

Reply:
There is a table within the letter that shows 7424 preprints with new results compared to the other categories (221 confirmatory and 105 contradictory).  Table is called: Descriptive Characteristics of Preprints in bioRxiv

Reply:
OMG the table doesn't show up in the web view but does in the PDF 🤯  😂